lib/util/conflicter.js

```diff
@@ -98,6 +98,34 @@ class Conflicter {
     async.series(this.conflicts.map(resolveConflicts), cb.bind(this));
   }
 
+  /**
+   * Check if the file contents at `filepath` conflict with the `contents` passed to the
+   * function
+   *
+   * If `filepath` points to a folder, we'll always return true.
+   *
+   * @param  {String} filepath Destination filepath (current with to compare with)
+   * @param  {String} contents The new content to compare with.
+   * @return {Boolean} `true` if there's a conflict, `false` otherwise.
+   */
+  _detectConflict(filepath, contents) {
```

Comment:
Why are we moving this into a method? I think it's clearer to use it directly as it's exactly the same function signature. This way reading the code you know right away it's using `detect-conflict`; no need to walk the methods up to know the implementation.

Reply:
My intention is to override detectConflict to use jsdiff (ignore indentation) instead.
But this is not the better way to do this, I will move to another PR.

Thanks.